assets/scss/base/_typography.scss

```diff
@@ -142,3 +84,44 @@ hr {
 i {
     position: relative;
 }
+
+// Links
+a {
+    color: inherit;
+    text-decoration: none;
+    transition: all 0.1s ease;
+
+    &:hover,
+    &:focus {
+        text-decoration: none;
+        cursor: pointer;
+        color: inherit;
+    }
+}
+
+// Text positioning
+.text-center {
+    text-align: center !important;
+}
+
+.text-right {
+    text-align: right !important;
+}
+
+.text-left {
+    text-align: left !important;
+}
+
+/**
+ * Helper classes for all sizes in the font size map. See the
+ * map in variables for a list of all sizes.
+ *
+ * Example:
+ * .text-sm, .text-md, .text-lg, .text-button
+ */
+
+@each $font-size-label, $font-size-map in $font-sizes {
```

Comment:
I'm thinking should we have if statements around any loops to prevent compiling from errors? At least in this case the helper methods would not be compiled if the `$font-sizes` variable doesn't exist. Also a nice convention to follow even when not using CSS.